src/documentation.html

```diff
@@ -48,7 +57,7 @@ <h2>Troubleshooting</h2>
   </div>
   <h1>Resources</h1>
   <div class="layout horizontal items">
-    <a href="/hardware">
+    <a href="/yellow-v1.2-standard">
```

Comment:
This is probably unintended change.

```suggestion
    <a href="/hardware">
```